config-connector/solutions/iam/kpt/member-iam/README.md

```diff
@@ -0,0 +1,84 @@
+Member IAM
+==================================================
+
+# NAME
+
+  member-iam
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant it a specific role (defaults to `compute.networkAdmin`) in the project.
+
+  You can also create any number of service accounts and grant them any number of roles by altering and applying the altered config YAMLs.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/member-iam member-iam
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a service account 
+  (`cnrm-system@[PROJECT_ID].iam.gserviceaccount.com`) with the following 
```

Comment:
A working Config Connector cluster depends on using the cnrm-system@[PROJECT_ID].iam.gserviceaccount.com service account... since we're not applying the role to this account, I don't think it's important to mention.

Reply:
In order to create a new service account in the desired project and apply the roles specified in the yaml file to the new service account, the cnrm-system@ gsa MUST have the following permissions. So that's a requirement. We don't assume the desired the project is the same as the project where the customer has installed KCC.

Reply:
That's why I think that explicitly calling out the service account name is confusing; which project is [PROJECT_ID] referring to?

Reply:
As we discussed offline, let's call it Config Connector "cnrm-system" service account.